### ISSUE TITLE
Error when generating a new instance

### ISSUE THREAD

**Issue by Mitchel703** [merschformann/RAWSim-O#12]
I have just installed the RAWSim-O framework and everything seemed to work fine. However when I run the RAWSimO.Visualization console and try to vreate a new instance I receive the following error:

![image](https://user-images.githubusercontent.com/119429794/204559191-04ba4522-660e-48df-9cfc-5765f36e8c8f.png)

This is the very first time running the project so all the parameters and settings are untouched. I have checked the project folder and the file reported in the error is present (~RAWSim-O-main\Material\Resources\Wordlists). Any clue how I can solve this problem?

**Comment by merschformann** [comment]
Yeah, it depends on from which directory you started the app. It checks all the folders listed in the dialog for presence of the file. So, it's sufficient to have the file in a `Resources` sub-folder of the current working directory.

Anyway, can you describe how you start the app? Maybe I can change something in the weird lookup logic to help make it more convenient for first time use. :relaxed:

**Comment by Mitchel703** [comment]
I started the app as follows: First I opened the RAWSim-O-Main folder with VS Code. Then I went in to the RAWSimO.Visualization directory using the command $cd ~RAWSim-O-main\RAWSimO.Visualization. After which I used the command $dotnet run to start the project. This opens up the RAWSimO console. 

There was indeed no resources folder in the directory from which I started the app, which was the ~RAWSim-O-main\RAWSimO.Visualization folder. So I just copied the directory ~RAWSim-O-main\Material\Resources and pasted it in to the RAWSimO.Visualization folder and this fixed the error. I am now able to run simulations so thank you for the tip!
